openmetadata-service/src/main/java/org/openmetadata/csv/EntityCsv.java

```diff
@@ -188,31 +190,31 @@ protected EntityInterface getEntityByName(String entityType, String fqn) {
   }
 
   protected final EntityReference getEntityReference(
-      CSVPrinter printer, CSVRecord record, int fieldNumber, String entityType, String fqn) throws IOException {
+      CSVPrinter printer, CSVRecord csvRecord, int fieldNumber, String entityType, String fqn) throws IOException {
     if (nullOrEmpty(fqn)) {
       return null;
     }
     EntityInterface entity = getEntityByName(entityType, fqn);
     if (entity == null) {
-      importFailure(printer, entityNotFound(fieldNumber, fqn), record);
+      importFailure(printer, entityNotFound(fieldNumber, fqn), csvRecord);
       processRecord = false;
       return null;
     }
     return entity.getEntityReference();
   }
 
   protected final List<EntityReference> getEntityReferences(
-      CSVPrinter printer, CSVRecord record, int fieldNumber, String entityType) throws IOException {
-    String fqns = record.get(fieldNumber);
+      CSVPrinter printer, CSVRecord csvRecord, int fieldNumber, String entityType) throws IOException {
+    String fqns = csvRecord.get(fieldNumber);
     if (nullOrEmpty(fqns)) {
-      return null;
+      return Collections.emptyList();
```

Comment:
We don't want to address this. Let's turn off this check. We have been returning null in many places including JSON payload no field instead of emptyList.

Reply:
sure